src/pages/development/components/app-server.md

```diff
@@ -192,3 +193,33 @@ public function _resetState(): void
     $this->data = [];
 }
 ```
+
+
+New in 2.4.8: An optional alternative to adding a _resetState method to a class has been added. Instead of adding _resetState to a class, a reset.json file may be added to a modules etc directory.  The reset.json file will define which properties of a class will be reset, and what they will be reset to after a request.
```

Comment:
```suggestion
<InlineAlert variant="info" slots="text"/>

The following section only applies to Adobe Commerce `2.4.8-beta`:

The `2.4.8-beta` includes an alternative to the `a _resetState` method. Instead of adding `_resetState` to a class, you can add a `reset.json` file to a module's `etc` directory.  The `reset.json` file defines which class properties are reset, and what they reset to after a request.
```